drivers/dma/dma_sam_xdmac.c

```diff
@@ -209,6 +209,23 @@ static int sam_xdmac_config(const struct device *dev, uint32_t channel,
 	dev_data->dma_channels[channel].data_size = data_size;
 	LOG_DBG("data_size=%d", data_size);
 
+	uint32_t xdmac_inc_cfg = 0;
```

Comment:
Didn't notice before, but `xdmac_inc_cfg` doesn't appear to be used? Or am I missing something?

Reply:
good catch, it should've been used when doing the channel_cfg. Needed for the spi tx/rx from a dummy value where there should be no increment